Client/components/Health.jsx

```diff
@@ -9,7 +9,7 @@ const Health = ({ health }) => {
   const card = <CreditCardIcon />;
```

Comment:
Not actually about this file, but looking at the README, the script seems to be wrong for seeding the db. 
It says npm run db:seed, rather than just npm run seed, like in package.json.

In Database/index.js, the line
`mongoose.connect(uri);`
should be
`mongoose.connect(uri, { useNewUrlParser: true, useUnifiedTopology: true });`
in order to avoid those deprecation warnings

Reply:
"npm run watch" should probably be in README, too.
I had to change the "Client" folder to lowercase c, "client" in order for the bundle to find it. (The C vs c is also a problem in the test files).
Resolve Public vs. public errors in webpack and server.